pcdsdevices/dccm.py

```diff
@@ -89,12 +172,11 @@ def energyToSi111BraggAngle(self, energy: float) -> float:
         Bragg angle: float
             The angle in degrees
         """
-        dspacing = 3.13560114
         energy = energy * 1000
-        bragg_angle = np.rad2deg(np.arcsin(12398.419/energy/(2*dspacing)))
+        bragg_angle = np.rad2deg(np.arcsin(np.float64(12398.419)/energy/(2*self.dspacing)))
         return bragg_angle
 
-    def thetaToSi111energy(self, theta):
+    def thetaToEnergy(self, theta):
```

Comment:
Is this theta the bragg angle? If so, we should make the function name consistent with `energyToBraggAngle`, that is, `braggAngleToEnergy`.